kythe/go/platform/tools/kzip/createcmd/createcmd.go

```diff
@@ -40,10 +41,10 @@ type createCommand struct {
 	rules  vnameRules
 
 	uri          kytheURI
-	source       repeatedStringSet
-	inputs       repeatedStringSet
+	source       flagutil.StringSet
+	inputs       flagutil.StringSet
 	hasError     bool
-	argument     repeatedString
+	argument     flagutil.StringList
```

Comment:
These are incompatible with the existing and intended semantics of these fields, particularly "argument".

While `source` and `inputs` may reasonably (but not necessarily) expected to not contain any embedded commas, the same cannot be said of `argument`.

Reply:
I reverted that flag.